### ISSUE TITLE
Validation loss calculation

### ISSUE THREAD

**Issue by yiqingguobuaa** [SKrisanski/FSCT#11]
Hi Sean, thanks for the great code!

I am looking at the following lines in the train.py script.

https://github.com/SKrisanski/FSCT/blob/af2b094a91c54763748a58983e5072ece936e878/scripts/train.py#L353-L354

For the calculation of validation loss and validation accuracy, should we divide the running loss by length of validation loader rather than that of train loader?

Cheers, Yiqing

**Comment by SKrisanski** [comment]
Hi Yiqing. 
Indeed it should. Thanks for drawing my attention to it! Pushing a fix now.
Cheers,
Sean
